### PR TITLE
Indicate configuration saving on LED Strip tab

### PR DESCRIPTION
**Pull request by ASDosjani**
image
Same as on VTX Tab, reduced delay from 2000 to 1500ms.

I'm using the same messages as on the vtx tab so I made the localization messages generic.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/led_strip.js
Comment (haslinghuis) [inline src/js/tabs/led_strip.js:588]: Repeated code if possible should probably go in /js/utils/common.js so we can easy extend to other tabs where applicable.

```comment
            function save_completed() {
                const buttonDelay = 1500;

                 // Allow firmware to make relevant changes before initialization
                setTimeout(() => {
                    saveButton.html(i18n.getMessage('buttonSaved'));

                    setTimeout(() => {
                        saveButton.html(oldText).removeClass('disabled');
                    }, buttonDelay);
                }, buttonDelay);
            }
```

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
